synchronizer/l1_sync_orchestration_test.go

```diff
@@ -11,7 +11,7 @@ type mocksOrgertration struct {
 	consumer *l1RollupConsumerInterfaceMock
 }
 
-func Test_A(t *testing.T) {
+func TestOrquestrationTodo(t *testing.T) {
```

Comment:
Is there an issue to implement it?

Reply:
No, basically is not done yet. And this PR is just a renaming, I'll do later to add the missing unittest